python/lsst/meas/base/colorUtilities.py

```diff
@@ -0,0 +1,43 @@
+# This file is part of meas_base.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Color Utilities such as extracting color information.
+"""
+
+import numpy as np
+from lsst.afw.image import Color
+
+__all__ = (
+    "colorExtractor",
+)
+
+
+def colorExtractor(record, columnNameValue="psf_color_value", columnNameType="psf_color_type"):
+    try:
```

Comment:
Are you sure you want this to fail silently? Do you need it to just return an empty color sometimes, or would it be better to raise an exception if the `columnNameValue` or `columnNameType` isn't in the record?